indico/modules/events/ical.py

```diff
@@ -235,6 +239,14 @@ def events_to_ical(
                 generate_event_component(event, user, organizer=organizer, skip_access_check=skip_access_check)
             ]
 
+        if reminder_minutes is not None:
+            for component in components:
+                alarm = icalendar.Alarm()
+                alarm.add('action', 'DISPLAY')
+                alarm.add('trigger', timedelta(minutes=-reminder_minutes))
+                alarm.add('description', component.get('description', f"Reminder: {component['summary']}"))
```

Comment:
Do we care to have this i18n'd?
```suggestion
                alarm.add('description', component.get('description', _("Reminder: {summary}").format(summary=component['summary'])))
```

Reply:
Line was exceeding 120 chars, I added a new reminder_text variable and and used `_( ` and `.format` notation